runtime/js/01_errors.js

```diff
@@ -134,6 +134,14 @@
       this.name = "NotSupported";
     }
   }
+  class WorkerError extends Error {
+    constructor(jsonString) {
+      const data = JSON.parse(jsonString);
+      super(data.error.message);
+      Object.assign(this, data);
+      this.name = "WorkerError";
+    }
+  }
```

Comment:
I'm not happy with this error - all errors defined in this file are available in `Deno.errors` but this one is not. Additionally I'm pretty sure that workers should throw plain `Error` or maybe even `DOMException`. @nayeemrmn thoughts?